src/ICS26Router.sol

```diff
@@ -212,7 +223,9 @@ contract ICS26Router is IICS26Router, IICS26RouterErrors, Ownable, ReentrancyGua
                 acknowledgement: msg_.acknowledgement,
                 relayer: _msgSender()
             })
-        );
+        ) {} catch {
+            // no-op
+        }
```

Comment:
we shouldn't actually be acking a packet if the callback cannot be made right? This feels wrong

Reply:
If the callback fails permanently then there's no point in resubmitting. This is for the case where ack just errors in an unrecoverable way. Resubmission won't fix anything here, so we may as well do cleanup.